internal/controller/pkg/resolver/reconciler.go

```diff
@@ -58,12 +58,12 @@ const (
 	errRemoveFinalizer      = "cannot remove lock finalizer"
 	errBuildDAG             = "cannot build DAG"
 	errSortDAG              = "cannot sort DAG"
-	errMissingDependencyFmt = "missing package (%s) is not a dependency"
+	errFmtMissingDependency = "missing package (%s) is not a dependency"
```

Comment:
No need to tackle it in this PR, but I like to keep the `errFmt` constants in a different block from the plain `err` constants. We've been talking lately about how errors with more context is better, which implies more `errFmt` is likely to be better. Keeping them in two blocks lets you quickly get an idea of how many errors in a file are the "better" `errFmt` style. e.g.:

```go
// Error constants
const (
    errFoo = ""
    errBar = ""

    errFmtFoo = ""
    errFmtBar = ""
)
```